### ISSUE TITLE
EKS v2 diff when public_access = true

### ISSUE THREAD

**Issue by rawmind0** [rancher/terraform-provider-rancher2#712]
When setting `public_access = true` on eks v2 clusters, getting this diff recurrently:

```
...
Terraform used the selected providers to generate the following execution plan. Resource actions are indicated with the following symbols:
  ~ update in-placeTerraform will perform the following actions:  # rancher2_cluster.foo will be updated in-place
  ~ resource "rancher2_cluster" "foo" {
        id                         = "c-xxxxxx"
        name                       = "foo"
        # (15 unchanged attributes hidden)
      ~ eks_config_v2 {
          ~ logging_types         = [
              - "api",
                "audit",
              + "api",
            ]
            name                  = "foo"
          ~ public_access_sources = [
              - "0.0.0.0/0",
            ]
            tags                  = {}
            # (9 unchanged attributes hidden)            # (2 unchanged blocks hidden)
        }
        # (1 unchanged block hidden)
    }
Plan: 0 to add, 1 to change, 0 to destroy.
...
```

To reproduce the issue, just use the docs example,   https://registry.terraform.io/providers/rancher/rancher2/latest/docs/resources/cluster#creating-eks-cluster-from-rancher-v2-using-eks_config_v2-for-rancher-v25x-or-above , and set `public_access = true`

**Comment by rawmind0** [comment]
When `public_access = true` the eks service seems to be adding `public_access_sources = ["0.0.0.0/0"]` by default but this field is not set as computed field at tfp side, so it's generating a diff, a real diff. As workaround, defining `public_access_sources = ["0.0.0.0/0"]` at tf file should suppress the diff.

The diff generated with `logging_services` is distinct. It's generated randomly when old and new values are compared, due to argument type []string is not sorted before it's being saved at tf state. As workaround, updating `logging_services` at tf file should suppress the diff.

Working on a fix for both issues.

**Comment by rawmind0** [comment]
Submitted PR #714 to fix the issue.

**Comment by rawmind0** [comment]
PR #714 already merged. The fix will be available at next tfp release, v1.17.0 

Please, reopen issue if needed.
